Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTip.cs

```diff
@@ -140,10 +140,10 @@ public string ToolTipText
         {
             set
             {
-                if (value != toolTipText)
-                {
-                    toolTipText = value;
-                    RefreshLocalContent();                   
+                toolTipText = value;
+                if (!Application.isPlaying)
+                {   // Only force refresh in edit mode
```

Comment:
I also see that only ToolTipText - is there a specific reason why only this property has this guard (but things like ContentScale, FontSize, and others don't have it?)

Reply:
Now that you point it out I guess I don't see the harm in changing them too.